corehq/apps/userreports/reports/view.py

```diff
@@ -312,6 +312,7 @@ def get(self, request, *args, **kwargs):
                     'is_static': self.is_static,
                     'error_message': error_message,
                     'details': details,
+                    'allow_delete': can_delete_report(request, self.spec),
```

Comment:
Hey @Charl1996 

Looks like this was added in the template here, https://github.com/dimagi/commcare-hq/commit/ca66a3012bbc3825aa0577a6edc6f3bf831d3e56 to hide the button.

Are you able to confirm its safe to add it back now? additionally we need to avoid this button for static reports?

Reply:
@mkangia From what I gather from the linked commit it looks like the intention was removing the button for the data registry reports. As for the `ConfigurableReportView` class, it wasn't added at any point, so I think it was an oversight (see [this commit](https://github.com/dimagi/commcare-hq/pull/11928/commits/3ed73355672d5c0c03317a60c99af2efb338002e) for when the button was first implemented).

That said, I do think I should still check for the report_id and whether or not it's a static report.

Reply:
Thanks @Charl1996